tests/sample.js

```diff
@@ -0,0 +1,8 @@
+import * as assert from 'assert';
+// const assert = require('assert');
```

Comment:
please clean code

Reply:
Done. Also merged with master before this change, please review latest commit.